python/doc/examples/meta_modeling/kriging_metamodel/plot_kriging_hyperparameters_optimization.py

```diff
@@ -258,31 +286,57 @@
 algo.run()
 result = algo.getResult()
 notUpdatedCovarianceModel = result.getCovarianceModel()
-notUpdatedCovarianceModel
+print(notUpdatedCovarianceModel)
 
 
 # %%
-def printCovarianceParameterChange(covarianceModel1,covarianceModel2):
+def printCovarianceParameterChange(covarianceModel1, covarianceModel2):
     parameters1 = covarianceModel1.getFullParameter()
     parameters2 = covarianceModel2.getFullParameter()
     for i in range(parameters1.getDimension()):
         deltai = abs(parameters1[i] - parameters2[i])
-        print("Change in the parameter #%d = %s" % (i,deltai))
+        print("Change in the parameter #%d = %s" % (i, deltai))
     return
 
 
 # %%
-printCovarianceParameterChange(covarianceModel,notUpdatedCovarianceModel)
+printCovarianceParameterChange(covarianceModel, notUpdatedCovarianceModel)
 
 # %%
 # We see that the parameters did not change *at all*: disabling the optimization allows to keep a constant covariance model. In a practical algorithm, we may, for example, add a block of 10 new points before updating the parameters of the covariance model. At this point, we may reuse the previous covariance model so that the optimization starts from a better point, compared to the parameters default values. This will reduce the cost of the optimization.
 
 # %%
-# Configure the optimization solver
-# ---------------------------------
+# Configure the local optimization solver
+# ---------------------------------------
+
+# %%
+# The following example shows how to set the local optimization solver.
+# We choose the SLSQP algorithm from NLOPT.
+
+# %%
+problem = solver.getProblem()
+local_solver = ot.NLopt(problem, "LD_SLSQP")
```

Comment:
Using the problem from a previous kriging is not required. We may as well use directly:
```Python
local_solver = ot.NLopt("LD_SLSQP")
```
This is because only the optimization class is required: the problem is created by the KrigingAlgorithm class.